package.json

```diff
@@ -67,8 +67,8 @@
     "dom-storage": "^2.1.0",
     "eslint": "^8.55.0",
     "express": "^4.18.2",
-    "global-jsdom": "^9.1.0",
-    "jsdom": "^23.0.1",
+    "global-jsdom": "^24.0.0",
```

Comment:
WOW -- big version bump!

Reply:
Sorry for not giving time for feedback, any concerns?